source/StellarObject.cpp

```diff
@@ -25,6 +25,17 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 using namespace std;
 
+namespace {
+	std::atomic<bool> usingMatches = false;
```

Comment:
This doesn't need to be atomic.

Reply:
(It's only ever used *after* it's been assigned to by at least the same thread)

Reply:
```suggestion
	bool usingMatches = false;
```